enterprise/apps.py

```diff
@@ -37,6 +39,9 @@ def ready(self):
         post_save.connect(handle_user_post_save, sender=self.auth_user_model, dispatch_uid=USER_POST_SAVE_DISPATCH_UID)
         pre_migrate.connect(self._disconnect_user_post_save_for_migrations)
 
+        # Segment initialization.
+        analytics.write_key = getattr(settings, 'LMS_SEGMENT_KEY')
```

Comment:
@UmanShahzad Do we need to do this here since we are already doing it in [LMS startup](https://github.com/edx/edx-platform/blob/master/lms/startup.py#L63)? This is causing an issue with running edxapp on a sandbox because during CMS startup this code runs and CMS settings do not contain an `LMS_SEGMENT_KEY` (instead it is `CMS_SEGMENT_KEY`). If we do need to do this here we should at least fail gracefully if `LMS_SEGMENT_KEY` does not exist.

Reply:
I think you can erase it actually, yeah.

Reply:
I've pushed the deletion now on this branch, too.